tests/mattermost-smoketest.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test, expect, Page } from "@playwright/test";
+
+export const randomSuffix = () => {
+  return Math.random().toString().substring(2, 6);
+};
+
+export const currentDateTime = () => {
+  return Date();
+};
+
+async function loginToMattermost(page: Page) {
+  await page.goto("https://chat.uds.dev/login");
+  await page.getByRole("link", { name: "View in Browser" }).click();
+  await page.getByRole("link", { name: "Gitlab Icon GitLab" }).click();
+  await page.getByLabel("Username or email").fill("doug");
+  await page.getByLabel("Password").fill("unicorn123!@#");
+  await page.getByRole("button", { name: "Log In" }).click();
+}
+
+async function setupWorkspace(page: Page) {
+  await page.getByPlaceholder("Organization name").fill("Unicorns");
+  await page.getByTestId("continue").click();
+  await page.getByRole("button", { name: "Skip" }).click();
+  await page.getByRole("button", { name: "Finish setup" }).click();
+  await page.waitForURL(/\/unicorns\/channels\/town-square/);
+  await page.locator("button").filter({ hasText: "7" }).click();
+  
+  const button = page.locator('role=button[name="Not now"]');
+  if (await button.isVisible()) {
+    await button.click();
+  }
+}
+
+async function sendMessage(page: Page) {
+  const messageBox = page.getByRole('textbox', { name: 'Write to Town Square' });
+  await messageBox.fill('Hello world!');
+  await messageBox.press('Enter');
+  await expect(page.locator('#post-list .post-message__text').last()).toContainText('Hello world!');
+}
+
+test("test mattermost login, init and message", async ({ page, context }) => {
+  await loginToMattermost(page);
+
```

Comment:
```suggestion
  await page.waitForURL(
    /\/preparing-workspace|\/unicorns\/channels\/town-square/,
  );
```

Above got dropped from the end of loginToMattermost(). It either needs to be the last step in that function or called here before setting finalUrl. I tested it in both places again locally and it works for both pre and post setting up the organization.

Reply:
i'm just committing this suggestion so the tests will pass...